kintone/cli-kintone
# build: update the license management scripts

## Why



We have some configuration about licensed which we don't need anymore.

## What



- add license-manager execution to pnpm scripts
- update CI workflows
- change the output path of the `NOTICE` file

## How to test



```
pnpm install

// Check license analyze
pnpm license:analyze

// Check the artifact generation
pnpm build:artifacts
```

(Check the `build` and `License` CI)

## Checklist
